packages/editor/src/lib/components/default-components/DefaultSpinner.tsx

```diff
@@ -1,7 +1,7 @@
 /** @public @react */
 export function DefaultSpinner() {
 	return (
-		<svg width={16} height={16} viewBox="0 0 16 16">
+		<svg width={16} height={16} viewBox="0 0 16 16" role="alert" aria-busy="true">
```

Comment:
is `role="alert"` appropriate for a loading spinner? from mdn:
> Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.

Reply:
good push, i re-read what best practices are on loading/spinner and saw that i was reading some outdated/refuted practices. i updated it to more accepted practices.